exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/SpanAdapter.java

```diff
@@ -172,7 +172,7 @@ static Status toStatusProto(SpanData.Status status) {
     // todo: Update this when the proto definitions are updated to include UNSET and ERROR
     Status.StatusCode protoStatusCode = Status.StatusCode.STATUS_CODE_OK;
     if (status.getCanonicalCode() == StatusCode.ERROR) {
-      protoStatusCode = Status.StatusCode.STATUS_CODE_UNKNOWN_ERROR;
+      protoStatusCode = Status.StatusCode.STATUS_CODE_ERROR;
```

Comment:
I think there's a little more we need to do here. See the comments in the proto:  https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/trace/v1/trace.proto#L239-L249

Reply:
When `status.getCanonicalCode()` is `StatusCode.UNSET`, the `protoStatusCode ` should be set to `Status.StatusCode.STATUS_CODE_OK`. Is that right?

Reply:
I think we're also supposed to be setting the `deprecated_code` as well, if I read those comments correctly.

Reply:
Can you help to confirm again? Thanks.

Reply:
looks good!